at_onboarding_flutter/lib/screen/at_onboarding_generate_screen.dart

```diff
@@ -45,7 +45,16 @@ class _AtOnboardingGenerateScreenState
       absorbing: _isGenerating,
       child: Scaffold(
         appBar: AppBar(
-          title: const Text('Setting up your account'),
+          title: Text(
+            'Setting up your account',
+            style: TextStyle(
+              color: Platform.isIOS || Platform.isAndroid
+                  ? Theme.of(context).brightness == Brightness.light
+                      ? Colors.black
+                      : Colors.white
+                  : null,
```

Comment:
What will be default color in Desktop platforms ?

Reply:
On desktop platforms, the default color is white, with the background color of the app.